docs/pages/jpscore/jpscore_operativ.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Operational models
+keywords: simulation
+tags: [jpscore, model]
+sidebar: jupedsim_sidebar
+folder: jpscore
+permalink: jpscore_operativ.html
+summary: Several operational models are implemented in jpscore. An operational model defines how pedestrians moves from one time step to the next.
```

Comment:
Shouldn’t here also be mentioned, that the models also define how the pedestrians interact with each other?

Reply:
done.